### PR TITLE
Remove health status filter grey outlines on hover

### PR DESCRIPTION
**Pull request by afiune**
### :nut_and_bolt: Description
A very quick change to remove the grey outline of the health status filter on hover.

### :+1: Definition of Done
Hover state style update: Remove the grey outlines for the background container for the five filters (total, critical, warning, OK and unknown) above the table and in the sidebar.

image
image

### :athletic_shoe: Demo Script / Repro Steps
Build and start the chef-ui-library.
Then, navigate to http://localhost:3334/molecules/chef-status-filter-group

### :chains: Related Resources
Jira: https://chefio.atlassian.net/browse/A2-842
Closes https://github.com/chef/automate/issues/472

### :white_check_mark: Checklist

- [ ] Necessary tests added/updated?
- [ ] Necessary docs added/updated?
- [x] Code actually executed?
- [x] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

File: components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss [hunk 1]
```diff
@@ -107,6 +107,7 @@ chef-status-filter-group {
     }
 
     &:hover {
+      box-shadow: 0px 0px 0px 0px;
```
Comment (danielsdeleo) [inline components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss]: based on how the rest of this file is written, it looks like `0px` is the right way to do this, but would like someone with more UI context to confirm :)

**Comment by afiune** [comment]
TL; DR; The EAS team has agreed that this is actually good to merge. Thanks team! 💯

tenor-242830107
